subprojects/launcher/src/test/groovy/org/gradle/launcher/cli/BuildActionsFactoryTest.groovy

```diff
@@ -170,14 +164,10 @@ class BuildActionsFactoryTest extends Specification {
     }
 
     void isInProcess(def action) {
-        assert action instanceof RunBuildAction
-        assert action.executer instanceof SetupLoggingActionExecuter
-        assert action.executer.delegate instanceof SessionFailureReportingActionExecuter
-        assert action.executer.delegate.delegate instanceof StartParamsValidatingActionExecuter
-        assert action.executer.delegate.delegate.delegate instanceof GradleThreadBuildActionExecuter
-        assert action.executer.delegate.delegate.delegate.delegate instanceof ServicesSetupBuildActionExecuter
-        assert action.executer.delegate.delegate.delegate.delegate.delegate instanceof ContinuousBuildActionExecuter
-        assert action.executer.delegate.delegate.delegate.delegate.delegate.delegate instanceof InProcessBuildActionExecuter
```

Comment:
What, you don't like `delegate.delegate.delegate.delegate.delegate.delegate`? :-P